src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -30,47 +31,148 @@
+                                   :padding-right "8px"
+                                   :grid-auto-flow "column"
+                                   :grid-gap "4px"}
+                      [:button {:position "relative"
+                                :margin "auto"
+                                :width "32px"
+                                :height "32px"
+                                :display "flex"
+                                :align-items "center"
+                                :background "transparent"
+                                :color (color :body-text-color :opacity-med)
+                                :transition "background 0.075s ease-in-out, filter 0.075s ease-in-out, color 0.075s ease-in-out"
+                                :justify-content "center"
+                                :border 0}
+                       [:&:before {:content "''"
+                                   :border-radius "1000em"
+                                   :z-index -1
+                                   :position "absolute"
+                                   :background (color :background-plus-1)
+                                   :inset "4px"}]
+                       [:&.close {:color "#fff"}
+                        [:&:before {:background "#E9541F" ;; Ubuntu close button background color
+                                    }]]
+                       [:&.minimize [:svg {:position "relative"
+                                           :top "5px"}]]
+                       [:svg {:font-size "12px"}]]
+                      [:&.theme-light [:button:hover {:filter "brightness(92%)"}]]
+                      [:&.theme-dark [:button:hover {:filter "brightness(180%)"}]]]]})
 
-(def app-header-style
+
+(def mac-app-header-style
   {:grid-area "app-header"
    :justify-content "flex-start"
    :background-clip "padding-box"
+   :background (color :background-color :opacity-high)
+   :color (color :body-text-color :opacity-med)
    :align-items "center"
    :display "grid"
    :position "absolute"
    :top "0"
-   :border-bottom [["1px solid" (color :border-color)]]
-   :backdrop-filter "blur(0.375rem)"
-   :background (color :background-color :opacity-high)
+   :backdrop-filter "blur(20px)"
    :right 0
    :left 0
    :grid-template-columns "auto 1fr auto"
    :z-index "1070"
    :grid-auto-flow "column"
-   :padding "0 0.75rem"
+   :-webkit-app-region "drag"
+   :padding-left "88px"
+   :padding-right "22px"
+   :height "52px"
+   :border-bottom [["1px solid " (color :body-text-color :opacity-lower)]]
+   ::stylefy/manual [[:svg {:font-size "20px"}]
+                     [:button {:justify-self "flex-start"
+                               :-webkit-app-region "no-drag"}]
+                     ["&.is-fullscreen" {:padding-left "22px"
+                                         :height "44px"}]
+                     ["button:not(:hover):not(.is-active)" {:background "transparent"}]]})
+
+
+(def win-app-header-style
```

Comment:
Is it worth it to create an abstract parent for all the 3 OS-specific styles? Some shared styles.

Reply:
refactored